### ISSUE TITLE
ZF2 fix #6725

### ISSUE THREAD

**Issue by GeeH** [zendframework/zend-validator#100]
This issue has been moved from the `zendframework` repository as part of the bug migration program as outlined here - http://framework.zend.com/blog/2016-04-11-issue-closures.html

___
Original Issue: https://api.github.com/repos/zendframework/zendframework/issues/6736
User: @Qronicle
Created On: 2014-10-06T13:07:21Z
Updated At: 2015-02-19T19:55:15Z
Body

___
### Comment
User: @Ocramius
Created On: 2014-12-24T00:47:53Z
Updated At: 2014-12-24T00:47:53Z
Body
Linking #6725

___
### Comment
User: @Ocramius
Created On: 2014-12-24T00:49:15Z
Updated At: 2014-12-24T00:49:15Z
Body
More tests specific to #6725 are required here.

___
### Comment
User: @weierophinney
Created On: 2015-02-19T19:55:15Z
Updated At: 2015-02-19T19:55:15Z
Body
@Qronicle ping... we need unit tests before we can merge.

___

**Comment by weierophinney** [comment]
This repository has been closed and moved to laminas/laminas-validator; a new issue has been opened at https://github.com/laminas/laminas-validator/issues/29.
